dynamic-demo-plugin/src/components/ExamplePage.tsx

```diff
@@ -71,7 +71,7 @@ export const ExamplePage: React.FC<{ title: string }> = ({ title }) => {
                 {t('Prometheus loading')}
               </Alert>
             )}
-            {!error && loaded && (
+            {!error && loaded && result && (
```

Comment:
`result` should always be defined when `!error && loaded`

The bug looks to me like the console is older version where `usePrometheusPoll` hook was returning `loading` instead of `loaded`. In which case the solution is to just use newer console